artemis/reporting/export/db.py

```diff
@@ -0,0 +1,102 @@
+import datetime
+import urllib
+from collections import defaultdict
+from typing import DefaultDict, Dict, List, Optional, Set
+
+from tqdm import tqdm
+
+from artemis.config import Config
+from artemis.db import DB
+from artemis.reporting.base.language import Language
+from artemis.reporting.base.report import Report
+from artemis.reporting.base.reporters import reports_from_task_result
+from artemis.reporting.blocklist import BlocklistItem, filter_blocklist
+from artemis.reporting.utils import get_top_level_target
+
+
+class ExportDBConnector:
+    """
+    A wrapper around DB that loads data and converts them to Reports.
+    """
```

Comment:
`ReportDB`?

Reply:
I am against, as the DB contains task results, not reports

Reply:
DataLoader